assignments/prototypes.js

```diff
@@ -34,16 +36,20 @@ Humanoid.prototype.destroy = function() {
   * should inherit destroy() from GameObject's prototype
 */
 function CharacterStats(info) {
-  this.healthPoints = info.age;
+  GameObject.call(this, info); //calls from gameObject
+  this.name = info.name;
+  this.healthPoints = info.healthPoints;
   this.name = info.name;
+  CharacterStats.prototype.takeDamage = function() {
+   console.log(this.name + ` took damage!!`);
+  }
```

Comment:
this function should be outside this function and below where you are pulling in the prototypes from GameObject. It looks like you had this before and changed it - why is that?